ladybug/src/main/java/nl/nn/ibistesttool/LadybugPipe.java

```diff
@@ -115,15 +117,15 @@ public PipeRunResult doPipe(Object input, IPipeLineSession session) throws PipeR
 
 		Collections.sort(reports, reportNameComparator);
 		long startTime = System.currentTimeMillis();
-		boolean reportGeneratorEnabledOldValue = testTool.getReportGeneratorEnabled();
+		boolean reportGeneratorEnabledOldValue = testTool.isReportGeneratorEnabled();
```

Comment:
Is deze method werkelijk veranderd in deze change?

Reply:
Zie reactie over getMessageWithResolvedVariables(reportRunner).